openapi/cosmos.yaml

```diff
@@ -786,94 +822,53 @@ components:
           items:
             $ref: '#/components/schemas/bcBiomedicalConceptCoding'
         _links:
-          $ref: '#/components/schemas/bcDataElementConceptsLinks'
+          $ref: '#/components/schemas/bcBiomedicalConceptLatestLinks'
+        dataElementConcepts:
+          type: array
+          items:
+            $ref: '#/components/schemas/bcDataElementConceptElement'
       required:
         - conceptId
         - category
         - shortName
         - definition
 
-    bcDataElementConceptsLinks:
+    bcBiomedicalConceptLatestLinks:
       type: object
       properties:
         self:
-          $ref: '#/components/schemas/bcDataElementConceptsRef'
+          $ref: '#/components/schemas/bcBiomedicalConceptLatestRef'
         parentBiomedicalConcept:
-          $ref: '#/components/schemas/bcBiomedicalConceptParentRef'
+          $ref: '#/components/schemas/bcBiomedicalConceptParentLatestRef'
         parentPackage:
-          $ref: '#/components/schemas/bcPackageRef'
-        dataElementConcepts:
-          type: array
-          items:
-            $ref: '#/components/schemas/bcDataElementConceptRef'
+          $ref: '#/components/schemas/bcPackageLatestRef'
 
-    bcDataElementConceptsRef:
+    bcBiomedicalConceptParentLatestRef:
       type: object
-      description: Data Element Concept Reference
+      description: Biomedical Concept Parent Reference
       properties:
         href:
           type: string
-          example: /mdr/bc/packages/2022-10-26/biomedicalconcepts/C49676/dataelementconcepts
+          example: /mdr/bc/packages/2022-10-26/biomedicalconcepts/C49672
         title:
           type: string
-          example: Pulse Rate Data Element Concept List
+          example: Vital Signs Measurement
         type:
           type: string
-          example: Data Element Concept List
+          example: Biomedical Concept
 
-    bcDataElementConceptRef:
+    bcPackageLatestRef:
       type: object
-      description: Data Element Concept Reference
       properties:
         href:
           type: string
-          example: /mdr/bc/packages/2022-10-26/biomedicalconcepts/C49676/dataelementconcepts/C123975
+          example: /mdr/bc/packages/2023-02-13/biomedicalconcepts
         title:
           type: string
-          example: Vital Signs Laterality (Pulse Rate)
+          example: Biomedical Concept Package Effective 2023-02-13
         type:
           type: string
-          example: Data Element Concept
-
-# Data Element Concept
-    bcDataElementConcept:
```

Comment:
Why was this deleted?

Reply:
When you look at https://wiki.cdisc.org/display/BIOMEDDEV/COSMoS+API+Endpoints there are several endpoints that were previously expected to be implemented, but are no longer deemed necessary (2 for BCs and 2 for SDTM specializations).
They were already specified in Swagger last year.
As part of this PBI they were deleted. That is what you see.

Reply:
Yes, but don't we still need dataElementConcepts?

Reply:
There is still bcDataElementConceptElement that is used by the endpoint that returns full data element concept content, as part of a BC. There is some duplication in the swagger that could indeed have been avoided.